packages/fxa-content-server/app/scripts/lib/experiment.js

```diff
@@ -25,7 +25,6 @@ const MANUAL_EXPERIMENTS = {
   // no special experiment is created.
   sendSms: BaseExperiment,
   signupPasswordCWTS: BaseExperiment,
-  tokenCode: BaseExperiment,
   totp: BaseExperiment,
```

Comment:
Looks like we don't have a totp experiment. Probably safe to remove!

Reply:
Great catch!